packages/create-app/templates/default-app/packages/app/src/sidebar.tsx

```diff
@@ -26,6 +27,7 @@ export const AppSidebar = () => (
     <SidebarDivider />
     {/* Global nav, not org-specific */}
     <SidebarItem icon={HomeIcon} to="./" text="Home" />
+    <SidebarItem icon={ExtensionIcon} to="api-docs" text="APIs" />
```

Comment:
I guess I completely broke this with my changes in #2739?

Reply:
Wrong # right? 😅

Reply:
@Fox32 You meant to link to another PR than that one right?

@stefanalund Just double checking, this doesn't need a leading slash?

Reply:
Yeah, I mean #2987 ...

Reply:
@freben I agree that it looks odd, but none of the other items has a leading slash: https://github.com/backstage/backstage/blob/master/packages/app/src/components/Root/Root.tsx#L84